document-store/src/integrationTest/java/org/hypertrace/core/documentstore/utils/Utils.java

```diff
@@ -99,14 +100,17 @@ public static Map<String, Object> convertDocumentToMap(Document document)
   }
 
   public static void assertDocsAndSizeEqual(
-      Iterator<Document> documents, String filePath, int expectedSize) throws IOException {
+      String dataStoreName, Iterator<Document> documents, String filePath, int expectedSize)
+      throws IOException {
     String fileContent = readFileFromResource(filePath).orElseThrow();
     List<Map<String, Object>> expected = convertJsonToMap(fileContent);
 
     int actualSize = 0;
     List<Map<String, Object>> actual = new ArrayList<>();
     while (documents.hasNext()) {
-      actual.add(convertDocumentToMap(documents.next()));
+      final Map<String, Object> doc = convertDocumentToMap(documents.next());
+      removeDateRelatedFields(dataStoreName, doc);
```

Comment:
Was this not a problem on the existing (or prior) test? or is this one not used everywhere?

Reply:
This was not used everywhere because this is required only when the selection list is empty.